src/Fantomas.Tests/UtilsTests.fs

```diff
@@ -112,3 +113,65 @@ SetupTesting.generateSetupScript __SOURCE_DIRECTORY__
 #endif
 """
     |> mergeAndCompare a b
+
+[<Test>]
+let ``when input is empty`` () =
+    let property (p: bool) : bool =
+        let before, after = List.partitionWhile (fun _ _ -> p) []
+        before = [] && after = []
+
+    Check.QuickThrowOnFailure(property true)
```

Comment:
You're not actually testing a property here, because there are no variables for FsCheck to generalise over. These should just be ordinary test cases, with `[<Test false>]` and `[<Test true>]`. Alternatively you could do `Check.QuickThrowOnFailure property` without supplying `true` and `false`.